packages/httpd.rb

```diff
@@ -23,29 +24,38 @@ class Httpd < Package
   depends_on 'libxcrypt'
   depends_on 'pcre'
   depends_on 'expat'
+  depends_on 'brotli' # R
+  depends_on 'glibc' # R
+  depends_on 'libnghttp2' # R
+  depends_on 'openssl' # R
+  depends_on 'pcre2' # R
+  depends_on 'util_linux' # R
+  depends_on 'zlib' # R
 
   print_source_bashrc
 
-  def self.build
-    system './configure',
-           "--prefix=#{CREW_PREFIX}/share/httpd",
-           "--libdir=#{CREW_LIB_PREFIX}",
-           "--bindir=#{CREW_PREFIX}/bin",
-           "--sbindir=#{CREW_PREFIX}/bin",
-           '--disable-maintainer-mode',
-           '--disable-debugger-mode'
-    system 'make'
+  def self.prebuild
+    ConvenienceFunctions.libtoolize('expat')
   end
 
-  def self.install
-    system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
-    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/etc/bash.d"
-    File.write "#{CREW_DEST_PREFIX}/etc/bash.d/10-httpd", <<~EOF
+  configure_build_extras do
+    File.write '10-httpd', <<~EOF
       #!/bin/bash
       [ -f #{CREW_PREFIX}/bin/apachectl ] && sudo #{CREW_PREFIX}/bin/apachectl start
     EOF
   end
 
+  configure_options "--prefix=#{CREW_PREFIX}/share/httpd \
+    --libdir=#{CREW_LIB_PREFIX} \
```

Comment:
We've already got this here: https://github.com/chromebrew/chromebrew/blob/master/lib/const.rb#L241

Reply:
Removed.